### PR TITLE
KEYCLOAK-9511 Update for EAP CD15

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.jboss.org/browse/KEYCLOAK-9511

This Pull Request rebases current SSO image on top of EAP CD 15. The main scope of the changes has been discussed in the [JIRA](https://issues.jboss.org/browse/KEYCLOAK-9511?focusedCommentId=13699747&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-13699747).

@iankko @luck3y Please take a look at it when you find a while.

### CODE REVIEW

**Comment by iankko** [comment]
@slaskawi LGTM AFAICT.

Have built both the standalone & OpenShift image locally, and both of them seem to be running fine, e.g.:

```
...
11:18:53,177 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0025: Red Hat Single Sign-On 7.3.0.GA (WildFly Core 7.0.1.Final-redhat-00001) started in 37824ms - Started 677 of 980 services (693 services are lazy, passive or on-demand)
```

But I am not sure WRT to those additional microprofile / observability related modules (`Prometheus`, `OpenTracing` & `Jeager` the PR containing also changes related with them):

- If we want to inherit these modules / functionality from EAP-CD-15 image, or
- If we want to handle these via separate (new) KEYCLOAK JIRA entry? (IOW for now we would only want minimal image, being able to run on top of EAP CD 15, but as much as possible close to upstream _standalone-ha.xml_ without those modules being included)

@stianst @mposolda Can you hint / suggest the right approach? 

Thank you, Jan

**Comment by mposolda** [comment]
Regarding the microprofile/jeager changes in standalone-openshift.xml, AFAIR our approach is to have the standalone-openshift.xml as close as possible to the standalone-ha.xml of the "standalone" keycloak distribution. So if the microprofile/jeager stuff is not in the standalone-ha.xml and if it is not needed for openshift, I think it shouldn't be added to standalone-openshift.xml as well IMO.

**Comment by slaskawi** [comment]
@iankko After having a small chat with @mposolda and @stianst we decided to remove metrics/tracing from this PR. I will send an update in a minute.

**Comment by iankko** [comment]
@slaskawi Ok, fair enough.

File: modules/sso/config/launch/setup/73/added/standalone-openshift.xml [hunk 1]
```diff
@@ -23,6 +23,7 @@
         <extension module="org.wildfly.extension.core-management"/>
         <extension module="org.wildfly.extension.elytron"/>
         <extension module="org.wildfly.extension.io"/>
+        <!-- ##TRACING_EXTENSION## -->
```
Comment (iankko) [inline modules/sso/config/launch/setup/73/added/standalone-openshift.xml]: @slaskawi Should this be removed too then?

Comment (slaskawi) [reply modules/sso/config/launch/setup/73/added/standalone-openshift.xml]: ooh yes! I'm on it!

Comment (slaskawi) [reply modules/sso/config/launch/setup/73/added/standalone-openshift.xml]: Done :) Thanks!

**Review by iankko — APPROVED** [review]
LGTM

- The PR is OK after review,
- Built the OpenShift image locally (with this PR) & seems to be working properly:

```
...
10:41:21,124 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0025: Red Hat Single Sign-On 7.3.0.GA (WildFly Core 7.0.1.Final-redhat-00001) started in 33189ms - Started 674 of 977 services (693 services are lazy, passive or on-demand)
```

=> ACK

**Pull request merged**